source/elements/oneDPL/source/parallel_api.rst

```diff
@@ -519,5 +519,41 @@ than an element in the range being searched.
 
 The elements of ``[start, end)`` must be partitioned with respect to the comparator used.
 
+.. code:: cpp
+
+    template <typename Policy, typename InputIt, typename Size, typename OutputIt>
+    OutputIt
+    histogram(Policy&& exec, InputIt start, InputIt end, Size num_intervals,
+              typename std::iterator_traits<InputIt>::value_type first_interval_begin,
+              typename std::iterator_traits<InputIt>::value_type last_interval_end,
+              OutputIt histogram_first); // (1)
+
+    template <typename Policy, typename InputIt1, typename InputIt2, typename OutputIt>
+    OutputIt
+    histogram(Policy&& exec, InputIt1 start, InputIt1 end, InputIt2 boundary_start,
+              InputIt2 boundary_end, OutputIt histogram_first);                             // (2)
```

Comment:
Does it make sense to align `// (1)` and `// (2)` so that both are well visible?

Reply:
yes, I'll fix this.